### PR TITLE
Fix BEMT induction factors (round 2)

### PR DESCRIPTION
**Pull request by tridelat**
- Use AeroDyn v15 calculation for induction factors
- Change induction factor bounds
- Change convergence criteria
- Deactivate tangential induction (convergence issue) for future release v0.1.0 (needs fix for future release)

### CODE REVIEW

File: src/aero/bemt.cpp [hunk 1]
```diff
@@ -88,24 +88,24 @@ chrono::ChVector2<double> seahowl::aero::get_induced_velocity(seahowl::aero::Bla
...
         // update induction factors
 
+        double tol_induction = 1e-6;  // tolerance for induction variables to avoid singularities
         // axial induction, based on AeroDyn v15 implementation
         double kk = element.chord_solidity * cn / (4.0 * loss_factor * pow(sin_phi, 2));
         if (kk <= 2.0 / 3.0) {
-            if (kk == -1.0) {
-                double temp = -aa_max * (1.0 + kk);
-                aa = (temp > 0.0) - (temp < 0.0);  // sign of temp
+            if (abs(kk + 1.0) < tol_induction) {
```
Comment (wenchaoyu) [inline src/aero/bemt.cpp]: is it better to use fabs than abs ?

File: src/aero/bemt.cpp [hunk 2]
```diff
@@ -119,15 +119,29 @@ chrono::ChVector2<double> seahowl::aero::get_induced_velocity(seahowl::aero::Bla
             double g2 = temp - (4.0 / 3.0 - ff) * ff;
             double g3 = temp - (25.0 / 9.0 - 2.0 * ff);
 
-            if (abs(g3) < 1e-6) {
+            if (abs(g3) < tol_induction) {
```
Comment (wenchaoyu) [inline src/aero/bemt.cpp]: change abs to fabs

**Review by wenchaoyu — CHANGES_REQUESTED** [review]

File: src/aero/bemt.cpp (hunk 1 above)
Comment (tridelat) [reply src/aero/bemt.cpp]: better to use fabs

**Review by wenchaoyu — APPROVED** [review]
looks good for me!

**Pull request merged**